internal/controller/util/recipe.go

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"github.com/ramendr/ramen/internal/controller/kubeobjects"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RecipeElements struct {
+	PvcSelector     PvcSelector
+	CaptureWorkflow []kubeobjects.CaptureSpec
+	RecoverWorkflow []kubeobjects.RecoverSpec
+	CaptureFailOn   string
+	RestoreFailOn   string
+}
+
+type PvcSelector struct {
```

Comment:
This is a duplicate of this in vrg_pvc_selector.go with another function getPVCSelector(). Need to check if the code can be simplified.